letters_count.py

```diff
@@ -0,0 +1,45 @@
+import string
+
+
+def count_letters():

```

Comment:
Please try to implement the function using this template: 
```
def count_letters(text):
   # here should be implementation
   return letter_dict
```
```
text (string) - Text itself where letters should be counted 
Example: "Python is an interpreted high-level programming language for general-purpose programming. Created by"

letter_dict (dictionary) - Map of letters and their count. 
Example:  {
                    "a" : 8,
                    "b" : 3,
                     ...
                 }
```

Reply:
Done